web/src/main/kotlin/no/nav/su/se/bakover/web/routes/sak/SakRoutes.kt

```diff
@@ -77,7 +106,7 @@ internal fun Route.sakRoutes(
         call.withSakId { sakId ->
             call.svar(
                 sakService.hentSak(sakId).fold(
-                    { NotFound.message("Fant ikke sak med id: $sakId") },
+                    { NotFound.errorJson("Fant ikke sak med id: $sakId", "fant_ikke_sak") },
```

Comment:
Finnes en felles errorJson for fant_ikke_sak i `Revurderingsfeilresponser.kt`, men den burde ligget i `Feilresponser.kt` (Men den har en mindre spesifikk message)